[DOCS-7711] removing TOC and on NDM data collected page/ restructuring

aliciascott opened the pull request:
<!-- *Note: Please remember to review the Datadog Documentation [Contribution Guidelines](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md) if you have not yet done so.* -->

### What does this PR do? What is the motivation?
[DOCS-7589](https://datadoghq.atlassian.net/browse/DOCS-7589)
this fix https://datadoghq.atlassian.net/browse/WEB-4740 fix caused some undesirable behavior, so , I disabled the TOC and moved the Events and Service checks to the top of the page. So re-opening this PR :) 

### Merge instructions
<!-- If you want us to merge this PR as soon as we've reviewed, check the box below. If you're waiting for a release or there are other considerations that you want us to be aware of, list them below. -->

- [x] Please merge after reviewing

### Additional notes
<!-- Anything else we should know when reviewing?-->

<!-- Previewing the PR: Assuming you are a Datadog employee and named your branch `<yourname>/<description>`, a preview build will run and links to the preview output will be auto-generated and posted in the PR comments. The links will 404 until the preview build is finished running. -->

[DOCS-7589]: https://datadoghq.atlassian.net/browse/DOCS-7589?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

brett0000FF commented:
@aliciascott - Heads up that the PR to fix responsive tables in shortcodes is merged. Do you still need this PR then?

aliciascott commented:
> @aliciascott - Heads up that the PR to fix responsive tables in shortcodes is merged. Do you still need this PR then?

oh my gosh, well no then !! LOL I guess I missed that or just missed the timing ! I will close this :)

aliciascott commented:
Re-opened this based on the un-expected behavior, I think we can still workaround this for now by disabling the TOC And I moved the other 2 sections to the top. Removed WIP label. Ready to review/merge
